content/actions/guides/building-and-testing-nodejs.md

```diff
@@ -37,17 +37,17 @@ We recommend that you have a basic understanding of Node.js, YAML, workflow conf
 
 {% data variables.product.prodname_dotcom %} provides a Node.js workflow template that will work for most Node.js projects. This guide includes npm and Yarn examples that you can use to customize the template. For more information, see the [Node.js workflow template](https://github.com/actions/starter-workflows/blob/main/ci/node.js.yml).
 
-To get started quickly, add the template to the `.github/workflows` directory of your repository.
+To get started quickly, add the template to the `.github/workflows` directory of your repository. (If your default branch is not named `main`, change `main` to your default branch's name.)
```

Comment:
I'd love feedback from a writer on how and whether to comment on what to do for people whose default branch is not named `main`.

Reply:
Whatever text ends up being used, the same text should appear in the Ruby version also changed in this commit.